src/components/CreateMemorandum/UploadMemorandum.vue

```diff
@@ -11,13 +11,15 @@
 
       <!-- Summary error message -->
       <div
-        v-if="!isCreateMemorandumValid"
+        v-if="!getCreateMemorandumStep.valid"
```

Comment:
~I see that you're trying to make it consistent with the prefix 'get' but I'm not sure if I like this '.valid' suffix... for me it looks like `getCreateMemorandumStep` will return an object that has a `valid` property.~

Reply:
Oh, nevermind...

Reply:
Haha, yes, that's exactly what it's doing :)